src/Omnipay/Realex/Message/DeleteCardRequest.php

```diff
@@ -71,6 +70,10 @@ protected function createResponse($data)
 
     public function getEndpoint()
     {
-        return $this->endpoint;
+            return $this->getParameter('AuthEndpoint');
```

Comment:
@3dgroup - sorry, another picky one, but could you adjust the indentation so it follows the PSR standard of 4 spaces.